stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2694,14 +2699,13 @@ final internal class _Native${Self}StorageOwner<${TypeParametersDecl}>
   }
 
   internal func bridgedAllKeysAndValues(
-    objects: UnsafeMutablePointer<AnyObject>,
-    _ keys: UnsafeMutablePointer<AnyObject>
+    objects: UnsafeMutablePointer<AnyObject>?,
+    _ keys: UnsafeMutablePointer<AnyObject>?
   ) {
     bridgeEverything()
     // The user is expected to provide a buffer of the correct size
     var i = 0 // Position in the input buffer
-    var position = 0 // Position in the dictionary storage
-    let capacity = bridgedNativeStorage.capacity
+    let count = bridgedNativeStorage.capacity
```

Comment:
Good catch. I think I misread the loops below and thought it actually *was* a count.